### DISCUSSION TITLE
kpanda/user-guide/gpu/gpu_matrix

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2836] [Announcements]
# kpanda/user-guide/gpu/gpu_matrix

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/gpu/gpu_matrix.html

**Comment by Ye-Ting** [comment]
天数 修改为  天数智芯

**Reply by lengrongfu** [reply]
收到

**Comment by wencanniu** [comment]
是否支持Nvidia Tesla T4 vGPU能力？

**Reply by lemontrees123** [reply]
支持的

**Comment by samzong** [comment]
建议增加 CUDA 版本和 CANN 的适配情况.

**Reply by lemontrees123** [reply]
通过 GPU operator安装的driver不需要关注cuda版本，会一把安装起来并且适配；若手动安装需要遵循NVIDIA官网的适配关系：https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#id5

**Reply by panpan0000** [reply]
Ubuntu 24.04 已经流行了，是否考虑加入support matrix
